File: engine/src/main/java/client/database/data/SkillMacroData.java
package client.database.data;

public class SkillMacroData {
   private int position;

   private int skill1Id;

   private int skill2Id;

   private int skill3Id;

   private String name;

   private int shout;

   public SkillMacroData(int position, int skill1Id, int skill2Id, int skill3Id, String name, int shout) {
      this.position = position;
      this.skill1Id = skill1Id;
      this.skill2Id = skill2Id;
      this.skill3Id = skill3Id;
      this.name = name;
      this.shout = shout;
   }

   public int getPosition() {
      return position;
   }

   public int getSkill1Id() {
      return skill1Id;
   }

   public int getSkill2Id() {
      return skill2Id;
   }

   public int getSkill3Id() {
      return skill3Id;
   }

   public String getName() {
      return name;
   }

   public int getShout() {
      return shout;
   }
}

File: engine/engine-model/src/main/scala/client/database/data/SkillMacroData.scala
package client.database.data

class SkillMacroData(private var _position: Int, private var _skill1Id: Int, private var _skill2Id: Int,
                     private var _skill3Id: Int, private var _name: String, private var _shout: Int) {
  def position: Int = _position

  def skill1Id: Int = _skill1Id

  def skill2Id: Int = _skill2Id

  def skill3Id: Int = _skill3Id

  def name: String = _name

  def shout: Int = _shout
}
